launcher/net/PasteUpload.cpp

```diff
@@ -8,8 +8,22 @@
 #include <QJsonDocument>
 #include <QFile>
 
-PasteUpload::PasteUpload(QWidget *window, QString text, QString url) : m_window(window), m_uploadUrl(url), m_text(text.toUtf8())
+std::array<PasteUpload::PasteTypeInfo, 4> PasteUpload::PasteTypes = {
+    {{"0x0", "https://0x0.st", ""},
+     {"hastebin", "https://hastebin.com", "/documents"},
+     {"paste (paste.gg)", "https://paste.gg", "/api/v1/pastes"},
+     {"mclogs", "https://api.mclo.gs", "/1/log"}}};
```

Comment:
(Suggestion) Maybe rename to "HasteBin", "Paste (paste.gg)" and "MCLogs" (appearance purposes)

Reply:
Hastebin isn't called HasteBin it's called Hastebin